update url-toolkit to ^2.1.3

tjenkinson opened the pull request:
See https://github.com/videojs/videojs-contrib-hls/pull/1029

I added the `^` given you are using a lockfile. If you want it to be static I can remove that.

forbesjo commented:
Looks like there are a lot of package-lock updates. When I removed and re-added your module to update/unlock I only noticed `url-toolkit` update (and `karma-detect-browsers` was removed). Could you update the module that way and overwrite this commit?

tjenkinson commented:
That's strange I just updated package.json and ran

npm install

Will have a look

On 26 Jan 2018, at 22:42, Joe Forbes <notifications@github.com<mailto:notifications@github.com>> wrote:


Looks like there are a lot of package-lock updates. When I removed and re-added your module to update/unlock I only noticed url-toolkit update (and karma-detect-browsers was removed). Could you update the module that way and overwrite this commit?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/videojs/videojs-contrib-hls/pull/1327#issuecomment-360912349>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ADG-WVFIF8IYW96-prPknJfQ6VjZJM9Eks5tOkbggaJpZM4Ru0JO>.

forbesjo commented:
Deleted my comment because I ran my commands against videojs-http-streaming and not this project. Currently testing this PR against a couple of sources and will merge shortly.

tjenkinson commented:
@forbesjo instead of tweaking package.json manually I ran `npm install url-toolkit@^2.1.3` and the changes are actually slightly less now

tjenkinson commented:
thanks!
